contrib/datadog/go.mod

```diff
@@ -39,7 +39,7 @@ require (
 	github.com/secure-systems-lab/go-securesystemslib v0.7.0 // indirect
 	github.com/stretchr/objx v0.5.2 // indirect
 	github.com/tinylib/msgp v1.1.8 // indirect
-	go.temporal.io/api v1.44.1 // indirect
+	go.temporal.io/api v1.45.1-0.20250319003343-cdea10847991 // indirect
```

Comment:
Can't merge until this is a stable tag and there is an integration test confirming this behavior even works